kuku/tests/test_values.py

```diff
@@ -39,13 +39,29 @@ def test_nested_dicts(values, expected):
     "values, expected",
     [
         (["a.0.b=v1", "a.0.c=v2"], {"a": [{"c": "v2"}]}),
-        # (["a.0.b=v1", "a.1.c=v2"], [{"a": [{"c": "v2"}]}, {"a": [{"b": "v1"}]}]),
+        (["a.0.b=v1", "a.1.c=v2"], {"a": [{"b": "v1"}, {"c": "v2"}]}),
```

Comment:
let's add a test for replacement here as well. Optionally we'd have an "invalid" index error test as well.

Reply:
I added both (a couple of lines below)

Reply:
I saw them, but those are for file replacement, not values.

Reply:
You are right. I added 2 new test cases.